### ISSUE TITLE
Images missing from balenahub

### ISSUE THREAD

**Issue by derhagen** [balena-labs-projects/balena-sound#622]
The readme states the following:
```
If you don't want to create a balenaCloud account you can deploy the app directly from [balenaHub](https://hub.balena.io/balenalabs/balenasound).
```
However, the link is broken and the respective images seem to be missing from balenahub.

**Comment by maggie0002** [comment]
You now need a balenaCloud account to deploy. Here is the link: https://hub.balena.io/organizations/balenalabs/apps/balenaSound

Could you raise a PR to update the docs?

**Comment by maggie0002** [comment]
The code is still open source, and still runs on exactly the same platform as before. Except now you get more privacy by deploying from your own account, where before you were attached to someone else's account, and more control over the updates. 

There is a fleet maintained by a community member if you prefer, but you are then tied to their version bumps: https://hub.balena.io/organizations/g_ema_alvarez/fleets/balena-sound

**Comment by maggie0002** [comment]
Before you were attached to someone else's account, who could see your devices, and potentially access your device. Not very secure. If you add it to your own account that can't happen. Only you can access it. More secure, more private. If you don't mind the potential security/privacy issues, then you can join this public fleet in the same was as before without needing an account: https://hub.balena.io/organizations/g_ema_alvarez/fleets/balena-sound

**Comment by maggie0002** [comment]
Good question, no that's not what I meant. The fleet was previously managed by balena, so your device was visible to balena. When joining any fleet, you place your device in the hands of fleet owners that can be seen on the Hub. Now that the fleet option has been removed, devices that were attached to that fleet are still able to run as before, are no longer visible by balena, but will also no longer receive automatic updates. 

The fleet option that remains I referred to is a community user, which balena has nothing to do with, and subsequently requires some consideration before joining, as is the case to any service that subscribes to automatic updates: https://hub.balena.io/organizations/g_ema_alvarez/fleets/balena-sound
